src/js/View.js

```diff
@@ -80,7 +80,12 @@ export let View = (function() {
                 'kernel': kernel,
                 'colormaps': colormaps,
             };
-            this.aladin.webglAPI = new Aladin.wasmLibs.webgl.WebClient(this.aladinDiv.id, shaders, resources);
+
+            try {
+                this.aladin.webglAPI = new Aladin.wasmLibs.webgl.WebClient(this.aladinDiv.id, shaders, resources);
+            } catch(e) {
+                alert("It may be possible that WebGL2 is not supported by default on your browser (Safari concerned).\nPlease enable it by checking:\nDeveloper Menu > Experimental Features > WebGL2 and reload the page.")
```

Comment:
I would make the message more explicite:

_WebGL2 is not supported by default in your browser. If you're using Safari, you can enable it by checking ..._

Reply:
added see my last commit